crates/sui-types/src/messages.rs

```diff
@@ -1005,7 +1117,7 @@ impl VerifiedTransaction {
         system_transaction
             .pipe(TransactionKind::Single)
             .pipe(|kind| {
-                TransactionData::new(
+                TransactionData::new_with_dummy_gas_price(
```

Comment:
Need to double check whether we should use dummy gas price here

Reply:
cc @emmazzz @lxfind We are using a `gas price === 1`(as opposed to the reference gas price) for Genesis and ChangeEpoch transaction, is this expected?

Reply:
We don't charge gas for those two transactions so it should be fine.